tests/src/selection.rs

```diff
@@ -108,23 +108,20 @@ fn test_selection_fail() {
     let mut context = Context::default();
     let contract_bin: Bytes = Loader::default().load_binary("selection");
     let out_point = context.deploy_cell(contract_bin);
-    let reward_contract_bin: Bytes = Loader::default().load_binary("reward");
-    let reward_out_point = context.deploy_cell(reward_contract_bin);
     let always_success_out_point = context.deploy_cell(ALWAYS_SUCCESS.clone());
 
     // prepare lock_args
     let always_success_lock_script = context
         .build_script(&always_success_out_point, Bytes::new())
         .expect("always_success script");
     let always_success_script_dep = CellDep::new_builder()
-        .out_point(always_success_out_point)
+        .out_point(always_success_out_point.clone())
         .build();
     let issue_lock_hash = always_success_lock_script.calc_script_hash();
     // prepare args for reward contract
     let reward_type_script = context
-        .build_script(&reward_out_point, Bytes::from(vec![1u8; 32]))
+        .build_script(&always_success_out_point, Bytes::from(vec![1u8; 32]))
```

Comment:
Why change it?

Reply:
to simplify the construction of selection's test. otherwise, more work need to be done to deal with reward script